### ISSUE TITLE
No module named 'ibeam.src.handlers'

### ISSUE THREAD

**Issue by BillWeld** [Voyz/ibeam#159]
**Describe the bug**

Traceback (most recent call last):
  File "./ibeam_starter.py", line 8, in <module>
    from ibeam.src.handlers.login_handler import LoginHandler
ModuleNotFoundError: No module named 'ibeam.src.handlers'

**To Reproduce**
Running standalone under Amazon Linux 2 on ARM architecture.

python3 ./ibeam_starter.py 

**Expected behavior**
Not sure but not this

**Environment**
IBeam version: 0.5.0
standalone:
Python version (standalone users only): Python 3.7.16
OS: 

  Operating System: Amazon Linux 2
       CPE OS Name: cpe:2.3:o:amazon:amazon_linux:2
            Kernel: Linux 5.10.192-182.736.amzn2.aarch64
      Architecture: arm64

**Additional context**
I doubt this can actually work in this setup--ibeam appears to need chrome or chromium and neither seems to be available for the ARM architecture.  All I have is an old Firefox.

**Suggest a Fix**
I'm not sure this is worth fixing from the developers' point of view.  I'd love to have it but I can live without it.

**Comment by Voyz** [comment]
Hey @BillWeld thanks for reporting this issue. There indeed seem to be some import issues. I'll look into it when I find some time. A temporary fix would be to modify the PYTHONPATH I'd imagine.

Also, are you sure chrome/chromium is unavailable on ARM? Just a naive search, but I'm seeing this:

https://community.arm.com/arm-community-blogs/b/tools-software-ides-blog/posts/native-chromium-builds-windows-on-arm
https://chromium.googlesource.com/chromium/src/+/main/docs/mac_arm64.md

**Comment by thunderwin** [comment]
facing same issue

**Comment by BillWeld** [comment]
Thanks @Voyz.  I haven't been able to find a chrome/chromium for Linux ARM.  I'm sure it's possible to build it but I haven't found a recipe for that either.

I don't know much about web tech but is it really necessary to have a full browser to convince the Client Portal to let you log in?  I would have thought some fancy script using curl could have done it.

**Comment by Voyz** [comment]
Hey @BillWeld @thunderwin thanks for bearing with me. The issue was due to rearranged imports in the ibeam_starter.py. There's a bit where we add currenet path to $PATH which ensures the imports work correctly:
```python
_this_filedir = os.path.abspath(os.path.dirname(__file__))
sys.path.insert(0, str(Path(_this_filedir).parent))
```

However my IDE (Intellij IDEA) keeps on rearranging the imports which I sometimes don't notice. I tried to fix this by putting that code in a function. Hope it stays this way. Let me know if that helps 👍

----
@BillWeld unfortunately yes, full browser is how we authenticate now. If you find a better solution to do it without a browser definitely let us know.
